src/Factory/ProductListViewFactory.php

```diff
@@ -162,7 +162,7 @@ private function getPriceView(PriceDocument $price)
     {
         /** @var PriceView $priceView */
         $priceView = new $this->priceViewClass();
-        $priceView->current = $price->getAmount();
+        $priceView->amount = $price->getAmount();
```

Comment:
The `current` name has been used with extending price field in mind. In the case, we would like to add an original price how the fields should be named? Anyway, the `PriceView` is also a part of an API of ShopApiPlugin and this change breaks it. Maybe the field name is a little bit unfortunate, but the change should be considered in the scope of whole ShopApiPlugin, not because of custom PriceDocument.